src/clr-angular/forms/common/error.ts

```diff
@@ -18,6 +20,17 @@ import { ControlIdService } from './providers/control-id.service';
     '[id]': 'controlIdService?.id + "-error"',
   },
 })
-export class ClrControlError {
-  constructor(@Optional() public controlIdService: ControlIdService) {}
+export class ClrControlError implements AfterViewInit {
+  constructor(
+    @Optional() public controlIdService: ControlIdService,
+    private ariaLiveService: AriaLiveService,
+    private el: ElementRef
+  ) {}
+
+  ngAfterViewInit() {
+    const textNode = this.el.nativeElement.childNodes[0];
```

Comment:
This will need a browser platform check `if (isPlatformBrowser(this.platformId)) {}`